Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:0.12.5
+RUN apt-get update && apt-get install -y --no-install-recommends python3-minimal python3-virtualenv python3-pip python3-dev python3-pil
+RUN python3 -m virtualenv -p /usr/bin/python3 --system-site-packages /var/www/shoop/venv
```

Comment:
OK then. I usually let PIL compile itself, but maybe installing it from deb packages is better idea for the Docker image.